addins/CFSample/customfunctions.json

```diff
@@ -342,6 +342,35 @@
       "options": {
         "stream": true
       }
+    },
+    {
+      "id": "customErrorReturn",
```

Comment:
The 'id' should match the first argument passed to `associate()`.

Reply:
Changed in the latest iteration.